src/components/HomePage.jsx

```diff
@@ -1,22 +1,65 @@
-import React, { Component } from 'react';
-import Image from "./assets/image.jpg";
+import React from 'react';
+import SearchIcon from "./assets/search.png";
+import LocationIcon from "./assets/location.jpg";
 import "bootstrap/dist/css/bootstrap.css";
+import '../styles/home.css'
 import "../index.css";
 
-class HomePage extends Component {
-  render() {
-    return (
-      <React.Fragment>
-        <div className="container">
+
+
+const HomePage = () => {
+  return ( 
+    <React.Fragment>
```

Comment:
I don't see the need for this fragment because we have one parent container div that embodies all other elements. We use a fragment when we're returning more than one element at the same level in the return statement.

Reply:
ok sir